tracing_internal_test.go

```diff
@@ -86,6 +88,41 @@ func TestSpanString(t *testing.T) {
 	assert.Equal(t, "TraceID=f,ParentID=0,SpanID=0", span.String())
 }
 
+func TestSetPeerHostPort(t *testing.T) {
```

Comment:
do we have tests for the following cases:
 - host:port with invalid port
 - no port (just host) -- something where `SplitHostPort` would fail